src/mainwindow.cpp

```diff
@@ -3918,10 +3920,7 @@ bool MainWindow::eventFilter(QObject *object, QEvent *event)
     }
     case QEvent::ActivationChange: {
         if (m_editorSettingsWidget->isHidden()) {
-            QApplication::setActiveWindow(
-                    this); // TODO: The docs say this function is deprecated but it's the only one
-                           // that works in returning the user input from m_editorSettingsWidget
-                           // Qt::Popup
+            QWidget::activateWindow();
```

Comment:
Nit: Please put this into a separate commit.

Reply:
done

Reply:
I'm not sure why you can't reproduce it, but here's a video if it helps:

https://github.com/user-attachments/assets/418ffeae-e4a6-4857-b3f1-81e3da589992

P.S.: I completely removed `~/.config/Awesomeness` prior to recording.

I also didn't do all steps because my terminal was already being spammed with `qt.sql.qsqlquery: QSqlQuery::value: not positioned on a valid record` at only step 2, and by then the db is already broken.